src/rpc/blockchain.cpp

```diff
@@ -2289,8 +2308,16 @@ static RPCHelpMan dumptxoutset()
     CAutoFile afile{file, SER_DISK, CLIENT_VERSION};
     NodeContext& node = EnsureAnyNodeContext(request.context);
     UniValue result = CreateUTXOSnapshot(
-        node, node.chainman->ActiveChainstate(), afile, path, temppath);
-    fs::rename(temppath, path);
+        format, node, node.chainman->ActiveChainstate(), afile, path, temppath);
+
+    switch (format) {
+    case UTXOSnapshotFormat::COMPACT:
+        fs::rename(temppath, path);
+        break;
+    case UTXOSnapshotFormat::SQLITE:
+        fs::remove(temppath);
```

Comment:
curious why we don't use the temppath for sqlite like we do on compact?

Reply:
`temppath` adds ".incomplete" to the end of the filename.
`path` is the path entered by the user.

`temppath` can eventually be removed from `CreateUTXOSnapshot()` as this parameter is used only once and for logging purposes. I think the same information can be obtained from `afile` as it is created from `temppath`.